types/application.yaml

```diff
@@ -3731,6 +3731,17 @@
     template:
     - application/pskc+xml
   registered: true
+- !ruby/object:MIME::Type
+  content-type: application/prql
+  friendly:
+    en: Pipelined Relational Query Language
+  encoding: base64
+  extensions:
+  - prql
+  xrefs:
```

Comment:
Please remove the `xrefs` as it is generally reserved for officially registered media types and `Maximilian_Roos` will not resolve properly. I also strongly recommend that the PRQL team [apply](https://www.iana.org/form/media-types) for a media type registration so that this can be made official.

Reply:
Okay, I have removed the `xrefs`.